config.default.yaml

```diff
@@ -193,6 +193,10 @@ sector:
   agriculture_machinery_electric_share: 0
   agriculture_machinery_fuel_efficiency: 0.7 # fuel oil per use
   agriculture_machinery_electric_efficiency: 0.3 # electricity per use
+  shipping_methanol: true
```

Comment:
Wouldn't it be better to make this like the `shipping_hydrogen_share` a `shipping_methanol_share`?